test/parallel/test-require-symlink.js

```diff
@@ -1,53 +1,80 @@
 // Flags: --preserve-symlinks
 'use strict';
 const common = require('../common');
+
+const { execSync, spawn } = require('child_process');
+
+if (common.isWindows) {
+  // On Windows, creating symlinks requires admin privileges.
+  // We'll only try to run symlink test if we have enough privileges.
+  try {
```

Comment:
I'm not too thrilled about this (I know it was like this before). I rather have an emperical test:
```js
const symLinkTest = path.join(common.tmpDir, 'symLinkTest .js');
try {
  fs.symlinkSync(__filename, symLinkTest);
} catch (e) {
  assert.strictEqual(e.code, 'EPERM');
  common.skip('insufficient privileges');
}
```

Reply:
Sure, done! (I changed the assert to an `if ... throw`.)

Reply:
Does this need to be tested upfront as opposed to wrapping the actual symlink calls in the test in a try-catch block?

Reply:
Hmmm...maybe even better: It looks like there's a `common.canCreateSymlink()` used in two files. I can use it here.